### DISCUSSION TITLE
How to use BlurTransformation.kt in version 2.1.0

### DISCUSSION THREAD

**Discussion by necatisozer** [coil-kt/coil#1362] [Q&A] [answered]
Question from [here](https://github.com/coil-kt/coil/issues/1350):

> I am using version 2.1.0, but the BlurTransformation has been removed. The documentation says that copying the file can be imported, but I have a lot of errors when I copy the code to my project. I don't know what package to import.
> 
> <img alt="截屏2022-07-07 19 23 19" width="985" src="https://user-images.githubusercontent.com/5456137/177762290-5a08dfa4-4446-4c19-a86b-2aee8025c8ee.png">

**Comment by guuilp** [comment, marked as answer]
Here you go: https://github.com/Commit451/coil-transformations/blob/main/transformations/src/main/java/com/commit451/coiltransformations/BlurTransformation.kt

It uses the deprecated RenderScript API, so if you want to update it, please check this: https://developer.android.com/guide/topics/renderscript/migrate

**Reply by necatisozer** [reply]
Thank you
